src/command/info/info.c

```diff
@@ -786,7 +811,17 @@ infoRender(void)
                         continue;
                     }
                     else
-                        strCatFmt(resultStr, "%s\n", INFO_STANZA_STATUS_OK);
+                    {
+                        // Get the specific lock part
+                        KeyValue *lockKv = varKv(kvGet(stanzaStatus, STATUS_KEY_LOCK_VAR));
+                        KeyValue *backupLockKv = varKv(kvGet(lockKv, STATUS_KEY_LOCK_BACKUP_VAR));
+
+                        // Change displayed status if lock is found
+                        if (varBool(kvGet(backupLockKv, STATUS_KEY_LOCK_BACKUP_HELD)))
+                            strCatFmt(resultStr, "%s (%s)\n", INFO_STANZA_STATUS_OK, strPtr(varStr(STATUS_KEY_LOCK_BACKUP_STR)));
+                        else
+                            strCatFmt(resultStr, "%s\n", INFO_STANZA_STATUS_OK);
+                    }
```

Comment:
This looks pretty good, but shouldn't we also get here if there are no backups but a backup is running? So instead of `status: error (no valid backups)` you'd get `status: ok (backup/expire running`?

At line 769, perhaps `if (!varBool(kvGet(backupLockKv, STATUS_KEY_LOCK_BACKUP_HELD)) && statusCode != INFO_STANZA_STATUS_CODE_OK))`?

Reply:
I'm not really feeling confortable with saying "ok" if we don't know (yet) that the backup will be correct. It would also give inconsistency between the JSON and the text output regarding the status.

I would rather "add" the information to the error message. I'll patch the code in that way.

If you prefer having "ok" even with JSON saying "error", I'll adjust the IF as you suggest.

Reply:
I like the way you did it, so time for tests!

Basically, what we need is to create a lock in infoTest.c at line 182 and 359. Then rerun the previous two tests at each location and make sure the JSON and text have been updated appropriately. Yes, this means recopying a lot of text, but that's how the tests work. Be sure to remove the lock right after your tests in each section. When the coverage report says 100% then you are done!

After the tests are done we'll talk about documentation.

Reply:
Great, Thanks for the review !

Should I rebase my branch before the tests part ?

Reply:
Rebasing often is generally a good idea, but I don't think this area of the code has changed in a while.

Reply:
Rebased, thanks.
I'll now have a look at the tests part :-)